src/analytics/models/monitoring_site.py

```diff
@@ -0,0 +1,198 @@
+import app 
```

Comment:
Could we also define the schemas from here? And move most of the business logic to `controllers` directory?